src/packages/core/resources/resource.controller.ts

```diff
@@ -1,10 +1,11 @@
+import { html } from '@umbraco-cms/backoffice/external/lit';
+import type { UmbControllerHost } from '@umbraco-cms/backoffice/controller-api';
+import { UmbControllerBase } from '@umbraco-cms/backoffice/class-api';
+import { UmbContextConsumerController } from '@umbraco-cms/backoffice/context-api';
```

Comment:
I think this happened because the eslint/prettier config had some trouble. If you run the formatted again on this file, it should correct itself back (`npm i && npx eslint --write src/packages/core/resources/resource.controller.ts`).